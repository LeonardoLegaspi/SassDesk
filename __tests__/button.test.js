var sass = require('node-sass');

describe('-----BUTTON TEST', ()=>{
    test('Buttons snapshot', async () => {
  
        var result = sass.renderSync({
          data: `
          @import './3_elements/button';
          `,
          outputStyle: 'nested'
        });
        
        expect(result.css.toString()).toMatchSnapshot();
      });
  })