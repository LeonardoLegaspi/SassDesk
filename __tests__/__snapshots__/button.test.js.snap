// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`-----BUTTON TEST Buttons snapshot 1`] = `
"/*
FIXME Is this useful? or limits to buttons with colors on Variables?
*/
.button, .button.tertiary-type {
  border: 2px solid #bfbfbf;
  border-radius: 5px;
  display: inline-flex; }

.button, .button.tertiary-type {
  cursor: pointer;
  background: transparent;
  color: #212120;
  font-size: 1rem;
  padding: 10px 30px;
  text-align: center;
  outline: none;
  transition: 300ms all; }
  .button.scheme {
    border: 2px solid #1165ED;
    border-radius: 5px;
    background-color: #1165ED;
    color: #fff; }
    .button.scheme:hover:not(.inverted) {
      border-color: #0e51bd;
      background: #0e51bd; }
      .button.scheme:hover:not(.inverted).empty {
        background: #fff;
        color: #1165ED; }
  .button.scheme-inverted {
    border-color: #1165ED;
    background: #fff;
    color: #1165ED; }
    .button.scheme-inverted:hover {
      color: #fff;
      background: #1165ED;
      border-color: #1165ED; }
  .button.success {
    border: 2px solid #46a93e;
    border-radius: 5px;
    background-color: #46a93e;
    color: #fff; }
    .button.success:hover:not(.inverted) {
      border-color: #378430;
      background: #378430; }
      .button.success:hover:not(.inverted).empty {
        background: #fff;
        color: #46a93e; }
  .button.success-inverted {
    border-color: #46a93e;
    background: #fff;
    color: #46a93e; }
    .button.success-inverted:hover {
      color: #fff;
      background: #46a93e;
      border-color: #46a93e; }
  .button.fail {
    border: 2px solid #d10000;
    border-radius: 5px;
    background-color: #d10000;
    color: #fff; }
    .button.fail:hover:not(.inverted) {
      border-color: #9e0000;
      background: #9e0000; }
      .button.fail:hover:not(.inverted).empty {
        background: #fff;
        color: #d10000; }
  .button.fail-inverted {
    border-color: #d10000;
    background: #fff;
    color: #d10000; }
    .button.fail-inverted:hover {
      color: #fff;
      background: #d10000;
      border-color: #d10000; }
  .button.warning {
    border: 2px solid #f1c40f;
    border-radius: 5px;
    background-color: #f1c40f;
    color: #212120; }
    .button.warning:hover:not(.inverted) {
      border-color: #c29d0b;
      background: #c29d0b; }
      .button.warning:hover:not(.inverted).empty {
        background: #fff;
        color: #f1c40f; }
  .button.warning-inverted {
    border-color: #f1c40f;
    background: #fff;
    color: #f1c40f; }
    .button.warning-inverted:hover {
      color: #212120;
      background: #f1c40f;
      border-color: #f1c40f; }
  .button.full-width {
    width: 100%;
    display: inline-block; }
  .button.small {
    font-size: 0.77rem; }
  .button.medium {
    font-size: 1.33rem; }
  .button.big {
    font-size: 1.66rem; }
  .button.primary-type {
    border: 2px solid #1165ED;
    border-radius: 5px;
    background-color: #1165ED;
    color: #fff; }
    .button.primary-type:hover:not(.inverted) {
      border-color: #0e51bd;
      background: #0e51bd; }
      .button.primary-type:hover:not(.inverted).empty {
        background: #fff;
        color: #1165ED; }
  .button.secondary-type {
    border-color: #bfbfbf;
    background: #fff;
    color: #212120; }
    .button.secondary-type:hover {
      color: #fff;
      background: #212120;
      border-color: #212120; }
  .button:hover {
    border-color: #595959; }
"
`;
